### DISCUSSION TITLE
Tangents direction in Wind Turbine Case

### DISCUSSION THREAD

**Discussion by dkrgajjela** [xcompact3d/Incompact3d#138] [Q&A] [answered]
Hello, I'm confused with the tangents **(turbine%blade(iblade)%tx(istation))**  direction that is considered in the code. Can someone explain how the positive and negative signs are considered in the cases **counterclockwise and clockwise** and also please share any reference where i can find this.
Thank You.

**Comment by nbeb** [comment, marked as answer]
Hello, please see the attached schematic, I hope it can help. Twist (pitch) can be thought to be provided to the code around the spanwise vector in the local coordinate system. Alternatively, it might be helpful to have a look at the thesis of G. Deskos (https://spiral.imperial.ac.uk/handle/10044/1/68512).

![schematic](https://user-images.githubusercontent.com/71822834/208119196-2cdd7ab2-79d4-40ac-95b7-a90aeab81930.jpeg)

**Reply by dkrgajjela** [reply]
Thank You @nbeb

**Reply by dkrgajjela** [reply]
@nbeb could you please tell whether the sign conventions remain same for the case of vertical axis wind turbines?

**Reply by dkrgajjela** [reply]
I mean, the tangent, normal and the span-wise vectors directions sign conventions
